src/components/apphost.js

```diff
@@ -140,6 +140,8 @@ define(["appSettings", "browser", "events", "htmlMediaHelper"], function(appSett
         (browser.tv || browser.xboxOne || browser.ps4 || browser.mobile) && features.push("physicalvolumecontrol");
         browser.tv || browser.xboxOne || browser.ps4 || features.push("remotecontrol");
         browser.operaTv || browser.tizen || browser.orsay || browser.web0s || browser.edgeUwp || features.push("remotevideo");
+        // might require isNativeApp check if any other issues surface
```

Comment:
Maybe we should add a link to the comments?.. to have more context in place
```suggestion
        // might require isNativeApp check if any other issues surface
        // see https://github.com/jellyfin/jellyfin-web/pull/51 for when it was probably broken
```

Reply:
I actually traced isNativeApp throughout the project, this shouldn't be an issue at all anymore. I would say we could even remove the first comment but I can leave that one just in case.

Basically, that variable is set in a single location in the entire codebase when `appMode` is set to either android or cordova, which will never return true in the bundled web interface from now on.